### ISSUE TITLE
Cannot continue if you don't enter a section title after a choice 

### ISSUE THREAD

**Issue by LemBurnstein** [ImpowerGames/impower#160]
**Describe the bug**
Cannot continue if you don't enter a section title after a choice 

**Steps To Reproduce**
1. engine
2. create a choice
3. delete the Section title for that choice

**Expected Result**
Game continues past that choice without skipping to a new section

**Actual Result**
Game Preview freezes

**Screenshots**
![image](https://user-images.githubusercontent.com/92612567/160260666-64686723-f7cb-4110-bebc-ed7131f65704.png)

**Comment by lovelle-cardoso** [comment]
@LemBurnstein I can't replicate this. Can you paste a script I can run in your bug report?

Here's what I tried and it seems to work:

```
# RootSection

BANG!

Again?

+ No more please > NoMore
+ Do it again! >

^

# NoMore

Ok let's move on.
```

**Comment by LemBurnstein** [comment]
# SAY_PLEBIANS

MEPHAN STOFAT
[[Mister Stomes]]
(WHISPERS)
PLEBIANS

AN ENTHUSIASTIC ORPHAN
[[Jon the Orphan]]
(SHOUTING OUT THE DOOR)
PLEBIANS

MEPHAN STOFAT
[[Mister Stomes]]
CLOSE THE DOOR URCHIN.

+ FOLLOW THIS STRANGE MAN INTO HIS APARTMENT > BAD_TIMES
+ CRY FOR HELP > 

^

# BAD_TIMES

AN ENTHUSIASTIC ORPHAN
[[Jon the Orphan]]
(SHOUTING OUT THE DOOR)
WHAT DOES THAT WORD MEAN MISTER STOMES?

MEPHAN STOFAT
(GRUFFLY)
SHUT UP POOR, AND BRING ME MY HEROINE. I NEED TO WATCH OVER SOME CONTRACTORS AT MY MIND PALACE. CALL THIS NUMBER IF I OVERDOSE.

**Comment by lovelle-cardoso** [comment]
@LemBurnstein Fixed by 927aee34

**Comment by LemBurnstein** [comment]
This works now (Also fixed the issue where if we had a prior choice menu preceding the choice with no section header, the repeat symbol would take you back to the first Section header after the prior choice menu instead of the beginning of the current section.
